tests/examples/wibox/container/conditional/extend.lua

```diff
@@ -0,0 +1,54 @@
+local parent = ... --DOC_HIDE --DOC_NO_USAGE
+local wibox  = require("wibox") --DOC_HIDE
+local awful = {screen = require("awful.screen")} --DOC_HIDE
+
+--DOC_HIDE Add a client and a tag
+local c = client.gen_fake {x = 45, y = 35, width=40, height=30} --DOC_HIDE
+client.focus = c --DOC_HIDE
+local t = require("awful.tag").add("Test", {screen=screen[1]}) --DOC_HIDE
+c:tags{t} --DOC_HIDE
+
+    local function extended_conditional()
+        return wibox.container.conditional {
+            argument_callback = function()
+                return client.focus, awful.screen.focused().selected_tag
+            end,
```

Comment:
So, this is an example for the `argument_callback`. Why not simply access `client.focus` and `awful.screen.focused().selected_tag` in the callback functions below directly? That would shrink the code of the widget by at least 10 lines.

Reply:
Answered above